src/modules/universe/RadixNode.ts

```diff
@@ -2,21 +2,25 @@ import { RadixNodeInfo } from '../..';
 import Long from 'long';
 
 export default class RadixNode {
+    public static SHARD_CHUNK_HALF_RANGE = Long.fromValue(1).shl(43)
+
     constructor(readonly nodeInfo: RadixNodeInfo, readonly wsAddress: string, readonly httpAddress: string) {}
 
     public canServiceShard(shard: Long): boolean {
```

Comment:
Is there any documented pseudocode for `canServiceShard` function somehere? :) Maybe in WP, Confluence? I don't think I've seen it.

Reply:
It's the same as `ShardSpace::intersects` in java, I have not generalised it

Reply:
Ah thx!